src/indexation/pht.cpp

```diff
@@ -25,39 +25,30 @@ namespace dht {
 namespace indexation {
 
 /**
- * Output the blob into string and readable way
+ * Get bit string representation of a blob for a specified length.
  *
- * @param bl   : Blob to print
+ * @param bl   A Blob
+ * @param len  The number of bits to consider.
  *
- * @return string that represent the blob into a readable way
+ * @return string that represent the blob
  */
-static std::string blobToString(const Blob &bl) {
+std::string Prefix::firstBitsToString(const Blob &bl, size_t len) const {
```

Comment:
@aberaud: In the case of `toString`, the ouput is the bit string up to `size_` bits since this is the most expected behavior. However, one could like to get the whole bit string of the prefix for debugging purposes. To be honest, this function is mostly being used in debugging purposes. That's why I left it this way.